### PR TITLE
Upgrade react 18

### PR DESCRIPTION
**Pull request by katie-day**
Upgrading to React 18, plus fixing a couple of tests.

### CODE REVIEW

File: project-template/src/Components/Quote/Quote.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ const Quote = ({
       <figcaption className={styles['quote__caption']}>
         {(quoteCitationText || quoteCitationUrl) && (
           <>
-            {quoteBy}
+            <span>{quoteBy}</span>
```
Comment (liamjohnston) [inline project-template/src/Components/Quote/Quote.tsx:22]: Why is this needed? To allow inbuilt stylability?

**Review by liamjohnston — APPROVED** [review]

File: project-template/src/Components/Quote/Quote.tsx (hunk 1 above)
Comment (katie-day) [reply project-template/src/Components/Quote/Quote.tsx:22]: @liamjohnston I couldn't get it to pass the tests without adding something around it. If you have any ideas on getting it to pass without the `span` I'd be happy to hear? 

I think having the `span` for stylability might be useful in some cases too.

Comment (liamjohnston) [reply project-template/src/Components/Quote/Quote.tsx:22]: Nope, that sounds fine :)
